rackcity/views/it_instance_views.py

```diff
@@ -167,3 +174,15 @@ def instance_page_count(request):
     instance_count = ITInstance.objects.all().count()
     page_count = math.ceil(instance_count / page_size)
     return JsonResponse({"page_count": page_count})
+
+
+def is_location_full(rack_id, elevation, height):
```

Comment:
I think it might be helpful to name this `instance_height` rather than `height`, I originally wasn't sure if it was instance height or rack height

Reply:
on it